README.md

```diff
@@ -166,6 +167,62 @@ mysql::db { 'mydb':
 }
 ```
 
+### Install Percona server on Centos
+
+This example shows how to do a minimal installation of a Percona server on a Centos (has been tested on Puppet 4.4 / Centos 7 / Percona Server 5.7) including the Percona server, client and bindings.
+
+The configuration change here is really minimal.
+
+**Note:** The installation of the yum repository is not part of this package and is here only to show a full example of how you can install this.
+
+```puppet
+yumrepo { 'percona':
+  descr    => 'CentOS $releasever - Percona',
+  baseurl  => 'http://repo.percona.com/centos/$releasever/os/$basearch/',
+  gpgkey   => 'http://www.percona.com/downloads/percona-release/RPM-GPG-KEY-percona',
+  enabled  => 1,
+  gpgcheck => 1,
+}
+
+class {'mysql::server':
+  package_name     => 'Percona-Server-server-57',
+  service_name     => 'mysqld',
```

Comment:
```
Service name on CentOS 6 has been renamed from mysqld back to mysql. This change requires manual service restart after being upgraded from Percona Server 5.7.10-1. Bug fixed #1542332.
```
From: https://www.percona.com/doc/percona-server/5.7/release-notes/Percona-Server-5.7.10-2.html

I'll update that. Thanks for the catch! :)